### ISSUE TITLE
training duck meets errors

### ISSUE THREAD

**Issue by diamond0910** [JiahuiLei/MoSca#11]
Thank you for your great work! I train the demo duck dataset and meeting the following error.

![error (1)](https://github.com/user-attachments/assets/9f923bbd-b765-4261-ad31-51e778c16e47)

Here are my scripts. Can you give me some advice?

`
CUDA_VISIBLE_DEVICES=1 python mosca_precompute.py --cfg ./profile/demo/demo_prep.yaml --ws ./demo/duck
`
`
CUDA_VISIBLE_DEVICES=1 python mosca_reconstruct.py --cfg ./profile/demo/demo_fit.yaml --ws ./demo/duck
`

My cuda version is 11.7, python version is 3.10.16, and here are my packages' version.
`
Package                          Version
-------------------------------- ---------------------
absl-py                          2.1.0
accelerate                       1.2.1
addict                           2.4.0
aiohappyeyeballs                 2.4.4
aiohttp                          3.11.11
aiosignal                        1.3.2
annotated-types                  0.7.0
antlr4-python3-runtime           4.9.3
anyio                            4.7.0
argcomplete                      3.5.2
argon2-cffi                      23.1.0
argon2-cffi-bindings             21.2.0
arrow                            1.3.0
asttokens                        3.0.0
async-lru                        2.0.4
async-timeout                    5.0.1
attrs                            24.3.0
babel                            2.16.0
beautifulsoup4                   4.12.3
black                            24.10.0
bleach                           6.2.0
blinker                          1.9.0
Brotli                           1.0.9
certifi                          2024.12.14
cffi                             1.17.1
charset-normalizer               3.3.2
chex                             0.1.82
chumpy                           0.70
click                            8.1.7
colorama                         0.4.6
comm                             0.2.2
ConfigArgParse                   1.7
contourpy                        1.3.1
cupy-cuda11x                     12.2.0
cycler                           0.12.1
Cython                           3.0.11
dash                             2.18.2
dash-core-components             2.0.0
dash-html-components             2.0.0
dash-table                       5.0.0
dataclasses                      0.8
debugpy                          1.8.11
decorator                        5.1.1
defusedxml                       0.7.1
diff_gaussian_rasterization      0.0.0
diff_gaussian_rasterization_add3 0.0.0
diffusers                        0.31.0
dm-haiku                         0.0.10
dm-tree                          0.1.8
docker-pycreds                   0.4.0
easydict                         1.13
einops                           0.8.0
einshape                         1.0
et_xmlfile                       2.0.0
etils                            1.11.0
evo                              1.30.4
exceptiongroup                   1.2.2
executing                        2.1.0
fastjsonschema                   2.21.1
fastrlock                        0.8.3
filelock                         3.13.1
Flask                            3.0.3
flax                             0.7.1
fonttools                        4.55.3
fqdn                             1.5.1
freetype-py                      2.5.1
frozenlist                       1.5.0
fsspec                           2024.12.0
fvcore                           0.1.5.post20221221
gdown                            5.2.0
gitdb                            4.0.11
GitPython                        3.1.43
glfw                             2.8.0
gmpy2                            2.1.2
gof_diff_gaussian_rasterization  0.0.0
grpcio                           1.68.1
h11                              0.14.0
httpcore                         1.0.7
httpx                            0.28.1
huggingface-hub                  0.27.0
hydra-core                       1.3.2
idna                             3.7
imageio                          2.25.0
imageio-ffmpeg                   0.5.1
importlib_metadata               8.5.0
importlib_resources              6.4.5
iopath                           0.1.10
ipykernel                        6.29.5
ipympl                           0.9.5
ipython                          8.30.0
ipywidgets                       8.1.5
isoduration                      20.11.0
itsdangerous                     2.2.0
jax                              0.4.13
jaxlib                           0.4.13+cuda11.cudnn86
jaxline                          0.0.5
jedi                             0.19.2
Jinja2                           3.1.4
jmp                              0.0.4
joblib                           1.4.2
json5                            0.10.0
jsonpointer                      3.0.0
jsonschema                       4.23.0
jsonschema-specifications        2024.10.1
jupyter                          1.1.1
jupyter_client                   8.6.3
jupyter-console                  6.6.3
jupyter_core                     5.7.2
jupyter-events                   0.11.0
jupyter-lsp                      2.2.5
jupyter_server                   2.14.2
jupyter_server_terminals         0.5.3
jupyterlab                       4.3.4
jupyterlab_pygments              0.3.0
jupyterlab_server                2.27.3
jupyterlab_widgets               3.0.13
kiwisolver                       1.4.7
kornia                           0.7.4
kornia_rs                        0.1.7
lightning-utilities              0.11.9
llvmlite                         0.43.0
lpips                            0.1.4
lz4                              4.3.3
Markdown                         3.7
markdown-it-py                   3.0.0
MarkupSafe                       2.1.3
matplotlib                       3.2.1
matplotlib-inline                0.1.7
mdurl                            0.1.2
mediapy                          1.2.2
mistune                          3.0.2
mkl_fft                          1.3.11
mkl_random                       1.2.8
mkl-service                      2.4.0
ml_collections                   1.0.0
ml_dtypes                        0.5.0
mmcv-full                        1.7.2
moviepy                          2.1.1
mpmath                           1.3.0
msgpack                          1.1.0
multidict                        6.1.0
munch                            4.0.0
mypy-extensions                  1.0.0
natsort                          8.4.0
nbclient                         0.10.2
nbconvert                        7.16.4
nbformat                         5.10.4
nest-asyncio                     1.6.0
networkx                         3.3
ninja                            1.11.1.3
notebook                         7.3.1
notebook_shim                    0.2.4
numba                            0.60.0
numexpr                          2.10.2
numpy                            1.26.4
omegaconf                        2.3.0
open3d                           0.18.0
opencv-contrib-python            4.6.0.66
opencv-python                    4.7.0.68
openpyxl                         3.1.5
opt_einsum                       3.4.0
optax                            0.1.4
orbax-checkpoint                 0.5.16
overrides                        7.7.0
packaging                        24.2
pandas                           2.2.3
pandocfilters                    1.5.1
parso                            0.8.4
pathspec                         0.12.1
pexpect                          4.9.0
Pillow                           9.5.0
pip                              24.2
platformdirs                     4.3.6
plotly                           5.24.1
plyfile                          1.1
portalocker                      3.0.0
proglog                          0.1.10
prometheus_client                0.21.1
prompt_toolkit                   3.0.48
propcache                        0.2.1
protobuf                         5.29.2
psutil                           6.1.1
ptyprocess                       0.7.0
pure_eval                        0.2.3
pycocotools                      2.0.8
pycparser                        2.22
pydantic                         2.10.4
pydantic_core                    2.27.2
pydot                            3.0.3
pyglet                           2.0.20
Pygments                         2.18.0
PyOpenGL                         3.1.0
pyparsing                        3.2.0
pyquaternion                     0.9.9
pyrender                         0.1.45
PySocks                          1.7.1
python-dateutil                  2.9.0.post0
python-dotenv                    1.0.1
python-json-logger               3.2.1
pytorch-lightning                2.0.2
pytorch3d                        0.7.8
pytz                             2024.2
PyWavelets                       1.8.0
PyYAML                           6.0.2
pyzmq                            26.2.0
referencing                      0.35.1
regex                            2024.11.6
requests                         2.32.3
retrying                         1.3.4
rfc3339-validator                0.1.4
rfc3986-validator                0.1.1
rich                             13.5.3
rosbags                          0.10.6
rpds-py                          0.22.3
Rtree                            1.3.0
ruamel.yaml                      0.18.6
ruamel.yaml.clib                 0.2.12
safetensors                      0.4.5
scikit-image                     0.19.0
scikit-learn                     1.6.0
scipy                            1.14.1
seaborn                          0.13.2
Send2Trash                       1.8.3
sentry-sdk                       2.19.2
setproctitle                     1.3.4
setuptools                       75.1.0
Shapely                          1.8.4
simple_knn                       0.0.0
six                              1.17.0
smmap                            5.0.1
sniffio                          1.3.1
soupsieve                        2.6
stack-data                       0.6.3
sympy                            1.13.3
tabulate                         0.9.0
tenacity                         9.0.0
tensorboard                      2.18.0
tensorboard-data-server          0.7.2
tensorboardX                     2.6.2.2
tensorstore                      0.1.71
termcolor                        2.5.0
terminado                        0.18.1
threadpoolctl                    3.5.0
tifffile                         2024.12.12
timm                             0.9.0
tinycss2                         1.4.0
tokenizers                       0.21.0
tomli                            2.2.1
toolz                            1.0.0
torch                            2.1.0
torch-geometric                  2.6.1
torchaudio                       2.1.0
torchmetrics                     1.6.0
torchvision                      0.16.0
tornado                          6.4.2
tqdm                             4.67.1
traitlets                        5.14.3
transformers                     4.47.1
transforms3d                     0.4.2
triangle                         20230923
trimesh                          3.12.6
triton                           2.1.0
types-python-dateutil            2.9.0.20241206
typing_extensions                4.7.1
tzdata                           2024.2
uri-template                     1.3.0
urllib3                          2.2.3
wandb                            0.19.1
wcwidth                          0.2.13
webcolors                        24.11.1
webencodings                     0.5.1
websocket-client                 1.8.0
Werkzeug                         3.0.6
wheel                            0.44.0
widgetsnbextension               4.0.13
wrapt                            1.17.0
xatlas                           0.0.9
xformers                         0.0.22.post7
yacs                             0.1.8
yapf                             0.43.0
yarl                             1.18.3
zipp                             3.21.0
zstandard                        0.23.0
`
